modules/boot/src/utils/hook.cpp

```diff
@@ -46,6 +46,8 @@ HOOK_DEF(void, onSwitch, (void*, int, int));
 
 int spawn_id_input = 0;
 bool g_flagLogEnabled = 0;
+int g_lastValidPoint = 0;
+int point = 0;
 
```

Comment:
this doesnt need to be a global variable. in the function just do `int point = ...`
its a local temp variable so you dont have to write `g_dComIfG_gameInfo.play.mStartStage.getPoint()` twice